cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationManager.kt

```diff
@@ -140,15 +142,24 @@ private constructor(
         result: TranslationResult,
     ): Set<LanguageFrontend<*, *>> {
         val usedFrontends = mutableSetOf<LanguageFrontend<*, *>>()
+
+        val externalSources: MutableList<File> = mutableListOf()
```

Comment:
Can we please have some doc here? What are these variables used for?

Reply:
Added doc